src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -1435,14 +1435,21 @@ def set_input(self, input_data):
         '''
         assert isinstance(input_data, AcquisitionData)
         self.input_data = input_data
-    def process(self):
+    def process(self, dcm_prefix=None):
         '''
         Processes the input with the gadget chain.
+        dcm_prefix: Python text string.
+        If dcm_prefix is not None, the reconstructed images are written to
+        files <dcm_prefix>_<image number>.dcm.
+        Otherwise, they are stored in memory and can be retrieved by
```

Comment:
Is it really "otherwise", i.e. will `get_output()` not work? that'd be a pity.

Reply:
Yes, unfortunately the set of gadgets in currently used Gadgetron 3.17 does not seem to allow sending images to both the Gadgetron client and DICOM files. I might try to edit `DicomFinishGadget` so that it sends ISMRMRD images to the client but, assuming I succeed, do we want to ship our modification of Gadgetron with SIRF?
To remind you, this PR is just a quick fix, for a proper one we need help from Gadgetron people, but we can only get it once we moved to the recent Gadgetron.

Reply:
ok. as long as it's clear in the documentation. Maybe let `get_output()` check if the prefix was set, and if so raise a runtime error that's somewhat informative.

Reply:
done